### PR TITLE
Update components for hybrids v7

### PR DESCRIPTION
**Pull request by smalluban**
hybrids v7 does not support arrays as a shorthand property value. In our case, we can simply replace it with `undefined` to skip transform, so the property will keep the value as it is set.

### CODE REVIEW

**Review by chrmod — APPROVED** [review]

File: packages/ui/src/styles/fonts.css [hunk 1]
```diff
@@ -203,51 +203,3 @@
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC,
     U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
 }
-/* roboto-300 - latin */
-@font-face {
```
Comment (chrmod) [inline packages/ui/src/styles/fonts.css:207]: why do we remove the fonts?

Comment (smalluban) [reply packages/ui/src/styles/fonts.css:207]: Generally, I recommend using system iOS/mac system font, as it supports all of the required chars, and it is the most consistent with the system UI. We can still provide our fonts. However, the definitions for the Roboto are broken, and for example, polish letters do not work, and they are taken from Open Sans, which looks ughly.

**Review by chrmod — APPROVED** [review]

**Comment by chrmod** [comment]
Please bump the version before the merge

**Comment by smalluban** [comment]
@chrmod ready for pushing to npm.

**Pull request merged**